vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -0,0 +1,132 @@
+import enum
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+import torch
+from torch.nn.parameter import Parameter
+
+from vllm.model_executor.layers.linear import (LinearMethodBase,
+                                               set_weight_attrs)
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig)
+
+
+class FP8Config(QuantizationConfig):
+    """Config class for FP8."""
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "fp8"
+
+    @classmethod
+    def get_supported_act_dtypes(cls) -> List[torch.dtype]:
+        return [
+            torch.bfloat16, torch.half, torch.float8_e4m3fn, torch.float8_e5m2
+        ]
+
+    @classmethod
+    def get_min_capability(cls) -> int:
+        return 90
```

Comment:
Add a note that 89 CUDA capability should work in the future but will require a pytorch upgrade https://github.com/pytorch/pytorch/blob/72271fb07ef88c33a5cb928eec6d8af8bf365493/aten/src/ATen/native/cuda/Blas.cpp#L825

Reply:
@comaniac Can you add the comment in the code as well?